<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="71993" started-at="2019-05-26T14:12:35Z" finished-at="2019-05-26T14:13:47Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="71993" started-at="2019-05-26T14:12:35Z" finished-at="2019-05-26T14:13:47Z">
      <class name="tests.TestSuite1">
        <test-method status="PASS" signature="sessionIntializing()[pri:0, instance:tests.TestSuite1@6e1567f1]" name="sessionIntializing" is-config="true" duration-ms="37894" started-at="2019-05-26T16:11:57Z" finished-at="2019-05-26T16:12:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- sessionIntializing -->
        <test-method status="PASS" signature="AdditionSuccessfulTC()[pri:0, instance:tests.TestSuite1@6e1567f1]" name="AdditionSuccessfulTC" duration-ms="34190" started-at="2019-05-26T16:12:35Z" finished-at="2019-05-26T16:13:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AdditionSuccessfulTC -->
        <test-method status="PASS" signature="MultiplySuccessfulTC()[pri:0, instance:tests.TestSuite1@6e1567f1]" name="MultiplySuccessfulTC" duration-ms="18840" started-at="2019-05-26T16:13:09Z" finished-at="2019-05-26T16:13:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MultiplySuccessfulTC -->
        <test-method status="FAIL" signature="SubtractSuccessfulTC()[pri:0, instance:tests.TestSuite1@6e1567f1]" name="SubtractSuccessfulTC" duration-ms="18935" started-at="2019-05-26T16:13:28Z" finished-at="2019-05-26T16:13:47Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [10.00 - 15.00 = 5.00] but found [10.00 - 15.00 = 25.00]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [10.00 - 15.00 = 5.00] but found [10.00 - 15.00 = 25.00]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:176)
	at org.testng.Assert.assertEquals(Assert.java:186)
	at tests.TestSuite1.SubtractSuccessfulTC(TestSuite1.java:33)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:882)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1189)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SubtractSuccessfulTC -->
      </class> <!-- tests.TestSuite1 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
